types/three/src/cameras/PerspectiveCamera.d.ts

```diff
@@ -148,6 +149,18 @@ export class PerspectiveCamera extends Camera {
      */
     getFilmHeight(): number;
 
+    /**
+     * Computes the 2D bounds of the camera's viewable rectangle at a given distance along the viewing direction.
+     * Sets minTarget and maxTarget to the coordinates of the lower-left and upper-right corners of the view rectangle.
+     */
+    getViewBounds(distance: number, minTarget: Vector2, maxTarget: Vector2): void;
+
+    /**
+     * Computes the width and height of the camera's viewable rectangle at a given distance along the viewing direction.
+     * Copies the result into the target Vector2, where x is width and y is height.
+     */
+    getViewSize(distance: number, minTarget: Vector2, maxTarget: Vector2): void;
```

Comment:
@Methuselah96 Here it should be

```ts
getViewSize(distance: number, target: Vector2): Vector2;
```

Reply:
Good catch, thanks. Fixed in https://github.com/three-types/three-ts-types/pull/800.